covid-19-support/src/components/ResultsList.vue

```diff
@@ -79,6 +75,16 @@ export default {
         this.$refs['results'].scrollTo(0, top)
       }
     }
+  },
+  methods: {
+    getClosedMessage: function (closedInfo) {
+      var day = closedInfo.day
+      if (this.selectedDay > 6 && day == this.today) {
+        return 'Closed Today'
+      }
+
+      return `Closed on ${this.$t(`dayofweek.${weekdaysJs[day].day}`)}`
```

Comment:
And so does this need to be added as a translatable string to the en.json language file :)

Reply:
Just noticed my original code didn't translate this bit, sorry :)